scoringrules/_crps.py

```diff
@@ -604,44 +604,45 @@ def crps_gamma(
     return crps.gamma(observation, shape, rate, backend=backend)
 
 
-def crps_normal(
+def crps_logistic(
     observation: "ArrayLike",
     mu: "ArrayLike",
     sigma: "ArrayLike",
     /,
     *,
     backend: "Backend" = None,
 ) -> "ArrayLike":
-    r"""Compute the closed form of the CRPS for the normal distribution.
+    r"""Compute the closed form of the CRPS for the logistic distribution.
 
     It is based on the following formulation from
-    [Geiting et al. (2005)](https://journals.ametsoc.org/view/journals/mwre/133/5/mwr2904.1.xml):
+    [Jordan et al. (2019)](https://www.jstatsoft.org/article/view/v090i12):
 
-    $$ \mathrm{CRPS}(\mathcal{N}(\mu, \sigma), y) = \sigma \Bigl\{ \omega [\Phi(ω) - 1] + 2 \phi(\omega) - \frac{1}{\sqrt{\pi}} \Bigl\},$$
+    $$ \mathrm{CRPS}(\mathcal{L}(\mu, \sigma), y) = \sigma \left\{ \omega - 2 \log F(\omega) - 1 \right\}, $$
 
-    where $\Phi(ω)$ and $\phi(ω)$ are respectively the CDF and PDF of the standard normal
-    distribution at the normalized prediction error $\omega = \frac{y - \mu}{\sigma}$.
+    where $F(\omega)$ is the CDF of the standard logistic distribution at the
+    normalized prediction error $\omega = \frac{y - \mu}{\sigma}$.
 
     Parameters
     ----------
     observations: ArrayLike
```

Comment:
The type hint suggestion can be removed in the docstrings. When `mkdocstrings` parses the code it will automatically infer the type from the function signature.

Reply:
(we will make sure to remove them from all other functions later)